english/covid-19/0/0-BBA.md

```diff
@@ -1,7 +1,7 @@
 Pick a rock, paper, or scissors?
 
-[Rock](../WIP.md)
+[🪨 Rock](../WIP.md)
```

Comment:
What is this sign mean?

Reply:
This is just a rock. Although the RPS is just a game, I thought having emojis for this is a creative idea.

Reply:
try to use this one 🥊